packages/template-ui/src/views/Home.vue

```diff
@@ -129,7 +130,19 @@ export default {
       else {
         // FIXME add API to query random content nodes
         console.debug(`Fetch ${this.carouselSlideNumber} random content nodes`);
-        this.loadingCarouselNodes = false;
+
+        window.kolibri.getContentByFilter({
+          extra: {
```

Comment:
Why this is called "extra" instead of adding to the current "options" attribute?